makeflow/src/dag_node.c

```diff
@@ -47,6 +47,7 @@ struct dag_node *dag_node_create(struct dag *d, int linenum)
 	n->resource_request = CATEGORY_ALLOCATION_FIRST;
 
 	n->umbrella_spec = NULL;
+	n->umbrella_logfile = NULL;
```

Comment:
With umbrella_spec we were planning (at some point) on the specification being unique to the node, are we also expecting umbrella_logfile to be unique to the node. If not this should only exist in the wrapper so the file names are consistent across all of the nodes.